src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -266,6 +267,40 @@ internal static OtlpMetrics.Metric ToOtlpMetric(this Metric metric)
                                 }
                             }
 
+                            var exemplars = metricPoint.GetExemplars();
+                            foreach (var examplar in exemplars)
+                            {
+                                if (examplar.Timestamp != default)
+                                {
+                                    byte[] traceIdBytes = new byte[16];
```

Comment:
Add TODOs to avoid allocating a new `byte[]` for TraceId and SpanId

Reply:
The behavior is same for Logs, Traces.
If we find a solution, we  can apply across all 3.